.github/workflows/build-cpp-windows-packages.yml

```diff
@@ -82,13 +82,18 @@ jobs:
           ./gradlew.bat dist -PcppPlatform=x64 -PcppConfiguration=Release
         working-directory: ice/java
 
+      # MSI packaging depends on VCInstallDir to locate The C++ merge modules
+      - name: Add VCInstallDir to Environment
+        run: |
+          $vsPath = & "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -latest -property installationPath
+          echo "VCInstallDir=$vsPath\VC" | Out-File -FilePath $env:GITHUB_ENV -Append
```

Comment:
https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions?tool=powershell#setting-an-environment-variable

according to this to latest format to use with PowerShell 6 is

```
"{environment_variable_name}={value}" >> $env:GITHUB_ENV
```

Reply:
Do we know if PowerShell 6 is the default with this runner?

Reply:
In other words actions we do 
echo "USING_BUILD_CACHE=true" >> $env:GITHUB_ENV
        echo "CLToolExe=C:\cl.exe" >> $env:GITHUB_ENV
        echo "CLToolPath=C:\" >> $env:GITHUB_ENV